fuzzers/005-tilegrid/generate_full.py

```diff
@@ -395,15 +395,10 @@ def propagate_IOI_Y9(database, tiles_by_grid):
     higher than the rest, just like for some of the SING tiles.
 
     """
-    arch = os.getenv('XRAY_DATABASE')
-    if arch in 'artix7':
-        tiles = ['RIOI3_X43Y9', 'LIOI3_X0Y9']
-    elif arch in 'kintex7':
-        tiles = ['LIOI3_X0Y9']
-    elif arch in 'zynq7':
-        tiles = ['RIOI3_X31Y9']
-    else:
-        assert False, "Unsupported architecture"
+    ioi_tiles = os.getenv('XRAY_IOI3_TILES')
```

Comment:
This should be changed to just keep of the pin function (e.g. avoid the PUDC), rather than an env var.

Reply:
The fact is that the PUDC IOB tiles are not connected to the ones that are expanded by one frame in the `propagate_IOI_Y9` function.

In the `pin_func.txt` file produced during the tiles extraction reports that the only IOB with the PUDC function is `IOB_X0Y44` (for artix 50T), while the IOI tiles with the extra frame address are `X0Y9` and `X43Y9`